website/docs/open-source/getting-started/deploy-OSS.mdx

```diff
@@ -138,6 +138,14 @@ This shows in the applications view with a yellow warning status indicating it i
 
 To resume updates, go back to the details page, click the resume button, and after a few seconds reconsolidation will continue.
 
+## Delete Podinfo 
+
+To delete Podinfo in the GitOps way, run this command:
+
+```
+flux/git cmd
```

Comment:
This is a placeholder and actually needs to be replaced with the actual command (for flux or git) that will delete podinfo.